### PR TITLE
Convert update_data.py to a function and call it from cli/query.py

### PR DESCRIPTION
**Pull request by mhmohona**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Related issue

Fixes - #159

### CODE REVIEW

**Comment by andrewtavis** [comment]
Great that this is already getting done, @mhmohona! Let us know when it's ready for a final review :)

**Comment by mhmohona** [comment]
This PR is up for review @andrewtavis @wkyoshida. It's my current result now - 

image

**Comment by andrewtavis** [comment]
Amazing, @mhmohona! Really looking forward to reviewing this 😊😊

File: src/scribe_data/cli/cli_utils.py [hunk 1]
```diff
@@ -122,4 +122,4 @@ def print_formatted_data(data: Union[Dict, List], data_type: str) -> None:
                 print(item)
 
     else:
-        print(data)
+        print(data)
```
Comment (andrewtavis) [inline src/scribe_data/cli/cli_utils.py:125]: For here, we do want the line at the end of the file. Should also be fixed by pre-commit :)

**Review by andrewtavis — APPROVED** [review]
Very minor changes were sent along in 9c791ee, @mhmohona 😊 Thanks so much for this! There's some real functionality in this CLI now! So great, and congratulations on a first major milestone! 👏🚀

**Pull request merged**